### PR TITLE
add venv support for half-life theme

### PR DESCRIPTION
**Pull request by slackr**

### CODE REVIEW

File: themes/half-life/half-life.theme.sh [hunk 1]
```diff
@@ -16,7 +16,13 @@ function _omb_theme_PROMPT_COMMAND {
   local ps_path="${_omb_prompt_green}\w${_omb_prompt_normal}"
   local ps_user_mark="${_omb_prompt_red}λ${_omb_prompt_normal}"
 
-  PS1="$ps_username in $ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
+  local python_venv
+  _omb_prompt_get_python_venv
+
+  PS1="$ps_username in $python_venv $ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
```
Comment (akinomyoga) [inline themes/half-life/half-life.theme.sh:22]:
```suggestion
  PS1="$ps_username in $python_venv$ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
```

File: themes/half-life/half-life.theme.sh [hunk 2]
```diff
@@ -16,7 +16,13 @@ function _omb_theme_PROMPT_COMMAND {
...
 }
 
+OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:-false}
+OMB_PROMPT_VIRTUALENV_FORMAT="${_omb_prompt_olive}(%s)${_omb_prompt_reset_color}"
```
Comment (akinomyoga) [inline themes/half-life/half-life.theme.sh:26]:
```suggestion
OMB_PROMPT_VIRTUALENV_FORMAT="${_omb_prompt_olive}(%s)${_omb_prompt_reset_color} "
```

File: themes/half-life/half-life.theme.sh (hunk 1 above)
Comment (akinomyoga) [reply themes/half-life/half-life.theme.sh:22]: With `in $python_venv $ps_path`, there will be double space when no python virtual environment is activated.

**Comment by slackr** [comment]
thanks, fix applied

**Review by akinomyoga — APPROVED** [review]

**Review by akinomyoga — APPROVED** [review]

**Comment by akinomyoga** [comment]
Thanks! I've merged it.

**Pull request merged**